x-pack/plugins/ml/public/application/trained_models/models_management/test_models/models/raw_output.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { Observable } from 'rxjs';
+import useObservable from 'react-use/lib/useObservable';
+import { MLJobEditor } from '../../../../jobs/jobs_list/components/ml_job_editor';
```

Comment:
As this wrapper around the Ace editor is now used in places other than for jobs, would be good to rename it in a follow-up.